submitted_models/marble_hd2_sensor_config_1/model.sdf

```diff
@@ -154,20 +154,57 @@
           <specular>1.0 1.0 1.0</specular>
           <pbr>
             <metal>
-              <albedo_map>model://MARBLE_HD2_SENSOR_CONFIG_1/materials/textures/HD2_Albedo.png</albedo_map>
-              <metalness_map>model://MARBLE_HD2_SENSOR_CONFIG_1/materials/textures/HD2_Metalness.png</metalness_map>
-              <roughness_map>model://MARBLE_HD2_SENSOR_CONFIG_1/materials/textures/HD2_Roughness.png</roughness_map>
+              <albedo_map>materials/textures/HD2_Albedo.png</albedo_map>
+              <metalness_map>materials/textures/HD2_Metalness.png</metalness_map>
+              <roughness_map>materials/textures/HD2_Roughness.png</roughness_map>
             </metal>
           </pbr>
           <!-- fallback to script if no PBR support-->
           <script>
-            <uri>model://MARBLE_HD2_SENSOR_CONFIG_1/materials/scripts/</uri>
-            <uri>model://MARBLE_HD2_SENSOR_CONFIG_1/materials/textures/</uri>
+            <uri>materials/scripts/</uri>
+            <uri>materials/textures/</uri>
             <name>HD/HD2_Diffuse</name>
           </script>
         </material>
       </visual>
 
+      <!-- thermal camera -->
+      <visual name="thermal_camera_visual">
+        <pose>0.435 -.01 0.125 0 0 0</pose>
+        <geometry>
+          <box>
+            <size>0.02 0.025 0.025</size>
+          </box>
+        </geometry>
+      </visual>
+      <sensor name="thermal_camera" type="thermal">
+        <pose>0.435 -.01 0.125 0 0 0</pose>
+        <camera name="thermal_camera">
+          <horizontal_fov>1.047</horizontal_fov>
```

Comment:
This might need to be re-calculated. According to the camera specs: `Field of view: 28.2 * 21.3`. However, I am not sure how to interpret this information (units?).